www/localPackage.ts

```diff
@@ -382,15 +396,27 @@ class LocalPackage extends Package implements ILocalPackage {
     public static getPackageInfoOrDefault(packageFile: string, packageSuccess: SuccessCallback<LocalPackage>, packageError?: ErrorCallback): void {
         var packageFailure = (error: Error) => {
             NativeAppInfo.getApplicationVersion((appVersionError: Error, appVersion: string) => {
-                if (appVersionError) {
-                    CodePushUtil.logError("Could not get application version." + appVersionError);
-                    packageError(appVersionError);
-                } else {
+                NativeAppInfo.getBinaryHash((binaryHashError: Error, binaryHash: string) => {
+                    /** 
+                     * For the default package we need the app version, 
+                     * and ideally the hash of the binary contents.
+                     */
+                    if (appVersionError) {
+                        CodePushUtil.logError("Could not get application version." + appVersionError);
+                        packageError(appVersionError);
+                        return;
+                    } 
```

Comment:
may be move `if (appVersionError) {...}` outside(before) of this `NativeAppInfo.getBinaryHash(...)` function call?